src/libOpenImageIO-C/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright 2008-present Contributors to the OpenImageIO project.
+# SPDX-License-Identifier: BSD-3-Clause
+# https://github.com/OpenImageIO/oiio/blob/master/LICENSE.md
+
+add_library(OpenImageIO-C SHARED 
+    c-imageio.cpp
+    c-imageio.h
+    c-imageio_defines.h
```

Comment:
I *think* that you don't need to list the .h files here in `add_library`, because they are not files to be compiled.

Reply:
If you have the IDE folder organization option turned on in CMake, listing the H files will cause them to appear in a Headers folder in the IDE. In Xcode at least, project searches are restricted to files actually referenced in an xcodeproj file, so IDE users probably prefer files being exhaustively listed like this.

Reply:
Ah, something I was not aware of! I thought this was just needless listing that had no effect. By all means, leave it be if it serves a purpose.

Aside: I've used Macs as my primary at-home and carry-around machines for the last 12 years and I've never run xcode.

Reply:
I was just copying what's done in libOpenImageIO :) (CMakeLists.txt, line 87)

Reply:
> I was just copying what's done in libOpenImageIO :) (CMakeLists.txt, line 87)

Aha, indeed you were.